Logic/ParkingActions.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Logic.SpacePark;
+using SpaceApi.Models;
+
+namespace Logic
+{
+    public class ParkingActions
+    {
+        public void Park(Starship ship, SpacePort portName)
+        {
+            var parkingCheck = new ParkingChecks();
+            StandardMessages.LoadingMessage();
+            var parkings = parkingCheck.ParkingLots(portName.PortName).Result;
+            var parkingsList = ArrayBuilder.ParkArray(parkings); // Create string array of all parkings to present to the user in the menu
+            Console.Clear();
+            var selectedOption = Menu.ShowMenu($"You have selected to park {ship.Name}. Choose parking spot", parkingsList);
+
+            Finish(parkings, selectedOption, ship); // Add ship to parking in database
+        }
+        private static void Finish(List<Parking> parkings, int index, Starship ship) // Final check before adding to database
+        {
+            var spaceApi = new SpaceParkApi();
+            var parkingCheck = new ParkingChecks();
+            if (parkingCheck.SizeTooBig(parkings, index, ship))
```

Comment:
Nested if stmts, can increase the readability of the code.
Rather then nesting if/else stmts, try either as the example below, or split it out in smaller functions
```
if (stmt) {
  return;
}

if (stmt2) {
  return;
}

...
```